src/components/EditDisplayName/EditDisplayName.module.scss

```diff
@@ -22,3 +22,10 @@
 .successMessage {
   margin-top: 8px;
 }
+
+.saveDisplayName {
+  &:hover:disabled {
+    cursor: default;
+    background-color: var(--btn-primary-bg);
+  }
```

Comment:
I noticed that the `Save` button was disabled but still changing the cursor and bg color on hover so I updated it!